internal/test/e2e/cloudwatch.go

```diff
@@ -0,0 +1,85 @@
+package e2e
+
+import (
+	"fmt"
+	"regexp"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/cloudwatch"
+
+	"github.com/aws/eks-anywhere/pkg/logger"
+)
+
+var svc *cloudwatch.CloudWatch
+
+func init() {
+	if s, err := session.NewSession(); err == nil {
+		svc = cloudwatch.New(s)
+	} else {
+		fmt.Println("Cannot create CloudWatch service", err)
+	}
+}
+
+func putInstanceTestResultMetrics(r instanceTestsResults) {
+	if svc == nil {
+		logger.Info("Cannot publish metrics as cloudwatch service was not initialized")
+		return
+	}
+
+	logger.Info("Publishing instance test result metrics")
+	erroredCount, failedCount, succeededCount := 0, 0, 0
```

Comment:
Could you add a comment about why these counts are always either 0 or 1?

Reply:
my pleasure